sdcm/cluster_k8s/__init__.py

```diff
@@ -616,19 +622,25 @@ def upgrade_scylla_cluster(self, new_version: str) -> None:
         self.replace_scylla_cluster_value("/spec/version", new_version)
         self.rollout_restart()
 
-    def update_scylla_config(self) -> bool:
+    def update_scylla_config(self):
         with self.scylla_yaml_lock:
-            if not self.scylla_yaml_update_required:
-                return False
+            scylla_yaml = self.scylla_yaml.copy()
+            # Remove parameters not allowed by scylla-operator
+            for param in ['seed_provider', 'rpc_address', 'cluster_name', 'listen_address']:
+                if param in scylla_yaml:
+                    del scylla_yaml[param]
+
             with NamedTemporaryFile("w", delete=False) as tmp:
                 tmp.write(yaml.safe_dump(self.scylla_yaml))
-            self.k8s_cluster.kubectl(f"create configmap scylla-config --from-file=scylla.yaml={tmp.name}",
-                                     namespace=self.namespace)
-            time.sleep(30)
-            self.rollout_restart()
+                tmp.flush()
+                try:
+                    self.k8s_cluster.kubectl(f"get configmap scylla-config", namespace=self.namespace, verbose=False)
+                    self.k8s_cluster.kubectl(f"delete configmap scylla-config", namespace=self.namespace)
```

Comment:
What about using `--ignore-not-found=true` flag for `kubectl delete` ?

Reply:
applied